azavea/nj-wildlife-habitat-vector-tile-map
# Add featured location cards to initial app view

### Overview
Some initial user feedback revealed that featured location cards may help a user that is not familiar with NJ zoom to a location quickly.

This adds featured location cards to the sidebar in the initial app state that fly-to specific areas.

### Demo

![2018-08-27 10 22 43](https://user-images.githubusercontent.com/19695900/44665196-55fb4100-a9e3-11e8-8557-b084cbef496a.gif)

### Testing
* git pull
* run localhost
* Go to http://localhost:8000/#
* Click on featured location title to zoom
* Draw area of interest
* Click `Draw New Area`
* Zoom in an out to see app state changes
